### PR TITLE
Adding JWT code and documentation

### PR DESCRIPTION
**Pull request by niio972**

### CODE REVIEW

File: src/config.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 //**********************************************************************************************
-//                                       config.php 
+//                                       config.php
```
Comment (Gouga34) [inline src/config.php:4]: why did you commit the file ? no modification has been done, remove the file update

File: src/WSModel.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 //**********************************************************************************************
-//                               WSModel.php 
+//                               WSModel.php
```
Comment (Gouga34) [inline src/WSModel.php:3]: why did you commited this file ? No code modification has been done here..

**Review by Gouga34 — CHANGES_REQUESTED** [review]
remove the committed files where no code has been updated

File: src/config.php (hunk 1 above)
Comment (niio972) [reply src/config.php:4]: The modification have been done by PHP CS Fixer . It finds an extra space after the filename and removes it.

File: src/WSModel.php (hunk 2 above)
Comment (niio972) [reply src/WSModel.php:3]: The modification have been done by PHP CS Fixer . It finds an extra space after the filename and removes it.

File: src/config.php (hunk 1 above)
Comment (Gouga34) [reply src/config.php:4]: ok for this modification than ;)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,198 @@
...
+}
+```
+
+
+
+## WSJWT (Web service JSON Web Token)
+
+### Key pair authentication
+
+A valid public need to be create key, if you already have a valid public key go to the next step **WSJWT implementation class example**
```
Comment (Gouga34) [inline README.md]: "A valid public need to be create key" --> A valid public **key** needs to be created

File: src/WSModel.php (hunk 2 above)
Comment (Gouga34) [reply src/WSModel.php:3]: ok for the spaces but not for : 
- the lines added 
- the function(arg)
{
} 

it does not follows the OpenSILEX dev rules

**Pull request merged**